cluster/gce/gci/configure-helper.sh

```diff
@@ -1876,6 +1875,7 @@ function prepare-konnectivity-server-manifest {
   params+=("--agent-service-account=konnectivity-agent")
   params+=("--kubeconfig=/etc/srv/kubernetes/konnectivity-server/kubeconfig")
   params+=("--authentication-audience=system:konnectivity-server")
+  params+=("--v=3")
```

Comment:
is this related?
can we get a better commit message?

Reply:
sorry this is not intended to be included in the PR. There were some issues with the tests that I'm trying to debug. Marking this as WIP.